fzf-tab.zsh

```diff
@@ -513,6 +530,25 @@ toggle-fzf-tab() {
     fi
 }
 
+fzf-tab-build-module() {
+  pushd $FZF_TAB_HOME/modules
+  CPPFLAGS=-I/usr/local/include CFLAGS="-g -Wall -O3" LDFLAGS=-L/usr/local/lib ./configure --disable-gdbm --without-tcsetpgrp
+  make -j
+  popd
+}
+
+if [[ -e $FZF_TAB_HOME/modules/Src/aloxaf/fzftab.so ]]; then
+  module_path+=("$FZF_TAB_HOME/modules/Src")
+  zmodload aloxaf/fzftab
+
+  if [[ $FZF_TAB_MODULE_VERSION != "0.1.0" ]]; then
+    print -P "%F{yellow}fzftab module needs to be rebuild%f"
+    fzf-tab-build-module
```

Comment:
I would probably put a prompt here to ask the user if they want to rebuild (since it can be a lengthy process).